test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -429,41 +433,22 @@ test.describe( 'Pattern Overrides', () => {
 						},
 					},
 				},
-				innerBlocks: [
-					{
-						name: 'core/heading',
-						attributes: { content: 'Outer heading (edited)' },
-					},
-					{
-						name: 'core/block',
-						attributes: {
-							ref: innerPattern.id,
-							content: {
-								[ paragraphName ]: {
-									content: 'Inner paragraph (edited)',
-								},
-							},
-						},
-						innerBlocks: [
-							{
-								name: 'core/paragraph',
-								attributes: {
-									content: 'Inner paragraph (edited)',
-								},
-							},
-						],
-					},
-				],
+				innerBlocks: [],
 			},
 		] );
-
-		await expect(
-			editor.canvas.getByRole( 'document', {
-				name: 'Block: Paragraph',
-				includeHidden: true,
-			} ),
-			'The inner paragraph should not be editable'
-		).toHaveAttribute( 'inert', 'true' );
+		// Check it renders correctly.
+		const headingBlock = editor.canvas.getByRole( 'document', {
+			name: 'Block: Heading',
+			includeHidden: true,
```

Comment:
Is this needed?

Reply:
I don't think so. Removed in [this commit](https://github.com/WordPress/gutenberg/pull/60721/commits/33bfb866f3d7995bab5218f72783bf3d0262b003).